src/pages/__tests__/test.title.js

```diff
@@ -3,7 +3,7 @@ import TitleMicro from 'components/MicroComp/titlemicro'
 import { shallow } from 'enzyme'
 import schema2 from 'components/Schema/basicschema2'
```

Comment:
Same here. Another note, we will want to also make sure each test has a test for when it should render null.